pages/pathways.jsx

```diff
@@ -0,0 +1,243 @@
+var React = require('react');
+var Header = require('../components/header.jsx');
+var Footer = require('../components/footer.jsx');
+var HeroUnit = require('../components/hero-unit.jsx');
+var SpacePathwayProfile = require('../components/space-pathway-profile.jsx');
+
+var PathwaysInfo = [
+  {
+    name: "Online Learning Done Face-to-Face",
+    type: "Independent Pathway",
+    description: 
+      "<p>Online learning offers tremendous opportunity, but often lacks a key component — the ability to collaborate with those around us. In this Pathway, we explore “Learning Circles,” or in-person study groups that complement online learning.</p>",
```

Comment:
We probably don't want to encode raw HTML, but instead use:
```
description: [
  "Online learning .... "
],
contacts: [
...
],
```
and then in our JSX we can generate:
```
<div>
{ description.map(function(text) { return <p>{text}</p>; }) }
</div>
```